docs/source/guides/getting_started/automating.rst

```diff
@@ -0,0 +1,135 @@
+Automating
```

Comment:
One thing I remembered is I have shared this automating link far and wide on tue internet. Changing it implies it will dead-link. Lets preserve the original name of the page

Reply:
Ah! I hadn't run into that issue in my Sphinx/RTD usage so far. I'm surprised that the only options for server-side HTTP 3## redirects are in the RTD project settings and can't be managed in the docs source. I'll revert the rename.